OPUNetTransportLayer.cpp

```diff
@@ -227,10 +229,10 @@ bool OPUNetTransportLayer::GetExternalAddress()
 	}
 
 	// Send the request packet to the game server (first port)
-	bool errorCode;
-	errorCode = SendTo(packet, gameServerAddr);
-	if (errorCode == false)
-		return false;		// Error
+	bool errorCode = SendTo(packet, gameServerAddr);

```

Comment:
I see you just couldn't resist. :wink:

Reply:
Yeah, I tried hard not to eliminate the forward declares of variables as I was working, but this one was too hard to pass up.